tests/numerics/fracture_deformation/test_fracture_propagation.py

```diff
@@ -582,7 +582,7 @@ class PropagationCriteria:
         values for u and parameters.
 
     """
-
+    @pytest.fixture(autouse=True)
```

Comment:
Should we be explicit about the scope of the fixture?

Reply:
I don't use it that way, but I don't know if I follow best practice.

Reply:
Let me rephrase: Have you considered what the scope of this fixture is? 
I think that it is `function`, i.e., it is invoked once per `test_...` method of this class. Do you agree? And is this the intended behaviour?

Reply:
No, I have not considered this.

This is a fix of functionality which in practice is not used, so honestly, I put a low priority on maintaining the tests beyond making sure they run, and do take too much time (this class runs pretty fast).